tests/e2e/local/minikube/install_prereqs_centos.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+# Check if yum is installed
+if ! yum --help > /dev/null; then
+    echo "yum not installed. Please install it manually and run this script again."
+    exit 1
+fi
+sudo yum --quite -y update
```

Comment:
I think you have a mispelling here (`-quite`) and it is really bad practice to update a person's development environment without:

1. warning them
2. providing them a mechanism to exit

Can this PR work without a yum update?

Reply:
Fixed now.

Reply:
A quick question: Do we need to remove `apt-get update` and `homebrew update` in other places?

cc @hklai for discussion.

Reply:
Discussed with @sdake, we have reached the agreement that we can keep the update but need to warn people what will happen.